### ISSUE TITLE
currentFunctionOnStack.name is undefined

### ISSUE THREAD

**Issue by mo22** [joshbtn/excelFormulaUtilitiesJS#46]
Hi,
when parsing a formula like A5 or =A1+B1 or anything that does not start with a function I get an error:

```
node_modules/excel-formula/src/ExcelFormulaUtilities.js:1117
                        switch (currentFunctionOnStack.name.toLowerCase()) {
                                                      ^

TypeError: Cannot read property 'name' of undefined

```

**Comment by joshbtn** [comment]
Hi mo22,

I'll Check this out.  Thanks.

**Comment by joshbtn** [comment]
This should now be fixed in the latest version 1.2.3
